src/components/footer.jsx

```diff
@@ -35,6 +35,7 @@ function Footer() {
       '\n',
     );
   }
+  // TODO: link underline is longer now after making it .map
```

Comment:
I actually just added link underline length as a mini-task in styling clean up. The lengths were also too long in mobile view in the footer